### PR TITLE
#27, #50: autoload included everywhere

### LINKED ISSUE #50: Proper use of Autoload everywhere
autoload should be used everywhere properly.
As in the composer.json I have specified the whole resources folder in classmap, that means YOU CAN ACCESS ANY CLASS IN THE WHOLE RESOURCES FOLDER BY JUST CALLING ` require $_SERVER['DOCUMENT_ROOT'] . '/../vendor/autoload.php'; ` in the heading of the file

### PR DESCRIPTION
**Pull request by AJV009**
included autoload wherever possible.
Fixes #50
MERGE https://github.com/JayKandari/bookhive/pull/49 before this so as to avoid possible conflicts (just my guess)

### CODE REVIEW

File: resources/src/books.php [hunk 1]
```diff
@@ -27,7 +26,7 @@ public function disp_book($recent)
             $stmt2 = $this->pdo->query('SELECT * FROM Book order by added_on desc limit 4');
          }
          $op=array();
-         while($row=$stmt2->fetch(PDO::FETCH_ASSOC))
+         while($row=$stmt2->fetch())
```
Comment (guptahemant) [inline resources/src/books.php]: Please verify once PDO::FETCH_ASSOC, You might need to pass this into fetch method.

File: resources/src/books.php [hunk 2]
```diff
@@ -38,9 +37,9 @@ public function search ($name)
          $stmt2 = $this->pdo->query('SELECT * FROM Book where title LIKE "%'.$name.'%"');
          $op=array();
          $row=0;
-         if ($stmt2->fetch(PDO::FETCH_ASSOC))
```
Comment (guptahemant) [inline resources/src/books.php]: Please verify once PDO::FETCH_ASSOC, You might need to pass this into fetch method.

File: resources/src/books.php [hunk 3]
```diff
@@ -38,9 +37,9 @@ public function search ($name)
...
+         if ($stmt2->fetch())
          {
-            while($row=$stmt2->fetch(PDO::FETCH_ASSOC))
+            while($row=$stmt2->fetch())
```
Comment (guptahemant) [inline resources/src/books.php]: same here

File: resources/src/books.php [hunk 4]
```diff
@@ -55,7 +54,7 @@ public function search ($name)
       public function edit()
       {
          $stmt2 = $this->pdo->query('SELECT * FROM Book');
-         while($row=$stmt2->fetch(PDO::FETCH_ASSOC))
+         while($row=$stmt2->fetch())
```
Comment (guptahemant) [inline resources/src/books.php]: same here

File: resources/src/books.php [hunk 5]
```diff
@@ -143,7 +142,7 @@ public function issue($bid,$id)
       {
          $sql1 = "SELECT available FROM Book WHERE Book.id='". $bid."'";
          $stmt1=$this->pdo->query($sql1);
-         $row1=$stmt1->fetch(PDO::FETCH_ASSOC);
+         $row1=$stmt1->fetch();
```
Comment (guptahemant) [inline resources/src/books.php]: same here

File: resources/src/books.php [hunk 6]
```diff
@@ -169,7 +168,7 @@ public function book_info($id)
       {
          $stmt2 = $this->pdo->query('SELECT * FROM book_user where id="'.$id.'"');
          echo "<table><th>Book Title</th><th>Issued On</th><th>Status</th>";
-         while($row2=$stmt2->fetch(PDO::FETCH_ASSOC))
+         while($row2=$stmt2->fetch())
```
Comment (guptahemant) [inline resources/src/books.php]: same here

File: resources/src/books.php [hunk 7]
```diff
@@ -181,7 +180,7 @@ public function book_info($id)
             }
             $stmt1 = $this->pdo->query('SELECT title FROM Book where Book.id="'.$row2["bid"].'"');
             
-            while($row1=$stmt1->fetch(PDO::FETCH_ASSOC))
+            while($row1=$stmt1->fetch())
```
Comment (guptahemant) [inline resources/src/books.php]: same here

Comment (AJV009) [reply resources/src/books.php]: @guptahemant  I have set it default in the config while declaring the PDO object in dbConn.php so it needed?

File: resources/src/books.php (hunk 1 above)
Comment (AJV009) [reply resources/src/books.php]: @guptahemant I have set it default in the config while declaring the PDO object in dbConn.php so it needed?

**Comment by AJV009** [comment]
@guptahemant I have set it default in the config while declaring the PDO object in dbConn.php so it needed?

**Comment by guptahemant** [comment]
hmm ok.

File: public-html/login.php [hunk 8]
```diff
@@ -1,11 +1,20 @@
...
 if (isset($_SESSION["logged_in"])) {
 	if ($_SESSION["admin"] == "admin") {
+=======
+require $_SERVER['DOCUMENT_ROOT'].'/../vendor/autoload.php';
+use db\user;
+if(isset($_SESSION["logged_in"]))
+{
+	if ($_SESSION["admin"]=="admin")
+	{
+>>>>>>> 45d5f71... autoload included everywhere
```
Comment (guptahemant) [inline public-html/login.php]: Resolve the conflicts here

**Review by guptahemant — CHANGES_REQUESTED** [review]

File: public-html/login.php (hunk 8 above)
Comment (AJV009) [reply public-html/login.php]: ooops why cant I see this in my local

Comment (AJV009) [reply public-html/login.php]: yeah fixed

**Comment by AJV009** [comment]
> hmm ok.

@guptahemant 
you just said "hmm ok" does that mean really OK or good / bad?

**Review by guptahemant — APPROVED** [review]

**Comment by guptahemant** [comment]
You will need to verify once, but i think it should work.

**Comment by AJV009** [comment]
> You will need to verify once, but i think it should work.

Oookay

**Pull request merged**